dependencies.gradle

```diff
@@ -0,0 +1,68 @@
+ext.dep = [
+  "kotlinStdLib": "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.2.51",
```

Comment:
it feels like our polyrepo impl is bleeding into misk here by requiring a json format for the gradle file

Reply:
A little bit. I think the upside of easier dependency management is worth the cost.